### ISSUE TITLE
Please make iperf3(Synocli ) start as service like iperf2 at DSM boot-up

### ISSUE THREAD

**Issue by once375ml** [SynoCommunity/spksrc#5378]
Although I wrote a schedule task to make it run at boot-up, please make it run as default.

Many thanks!

**Comment by once375ml** [comment]
Although I wrote a schedule task to make it run at boot-up, please make it run as default.

Many thanks!

**Comment by hgy59** [comment]
We created to synocli-* packages to provide command line utilities. 
It is not intended that such packages run any service.

As IMHO iperf* services must run as root, it is not an option to create dedicated packages for iperf2 and iperf3, as DSM >= 7 avoids to run services from thirdpary packages as root,

So the start of such services by a custom task is the only solution anyway

_probably we could enhance the diyspk/iperf2/3 packages to run as service, but this would be limited to DSM 6.x, And we do not publish such packages, you woutld have to built it by yourself._

**Comment by once375ml** [comment]
Thanks for you explanation! I am not an IT man. That means I donn't know Synology(DSM) developing policy well.

I also noticed that after the installation of Synocli Monitor Tools, iperf2 is running in background without any manual startup.

I wondering if iperf2 works that way, why can't iperf3.

**Comment by hgy59** [comment]
@once375ml if iperf2 is running, this is not the one of the synocli-monitor package, as no synocli package starts any service.

Please regard that the synogear package of synology contains iperf and iperf3 for DSM 6 and iperf3 for DSM 7.
see our wiki on [synogear](https://github.com/SynoCommunity/spksrc/wiki/FAQ-synogear). Maybe that synogear runs iperf2 upon installation.

**Comment by once375ml** [comment]
@hgy59 I am confused. I newly installed DSM 7.1 on DS118. The synocli-monitor tools is the first and only 3rd party package I installed. Then iperf2 is running in background. I swear I didn't run iperf2 in command line manually and I even don't know Synogear until you told me 10 mins ago

By the way, I checked the process with command Top and Htop, but I can't find any iperf2 process. I can only find a process named "iperf3", which is manually started by me. Weird!
